internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
+
+func LoadConfiguration() (Configuration, error) {
+	config := Configuration{}
```

Comment:
The `LoadConfiguration` function does not handle the case where the environment variables are set but empty. Although you used `notEmpty` in tags, it might be beneficial to explicitly check for empty values and return a custom error message indicating that those specific variables are required.